Sign "~" is incorrectly placed in command (#94038)
openshift/openshift-docs: modules/persistent-storage-csi-vsphere-top-aware-infra-top.adoc

{product-title} recommends using the infrastructure object for specifying failure domains in a topology aware setup. Specifying failure domains in the infrastructure object and specify topology-categories in the `ClusterCSIDriver` object at the same time is an unsupported operation.
====

.Procedure

> . In the VMware vCenter vSphere client GUI, define appropriate zone and region catagories and tags.

Reviewer: ```suggestion
. In the VMware vCenter vSphere client GUI, define appropriate zone and region categories and tags.
```

+

> While vSphere allows you to create categories with any arbitrary name, {product-title} strongly recommends use of `openshift-region` and `openshift-zone` names for defining topology.

Reviewer: ```suggestion
Although vSphere allows you to create categories with any arbitrary name, {product-title} strongly recommends using `openshift-region` and `openshift-zone` names for defining topology.
```
As per IBM style guide:
>Use while only to refer to a period of time. To avoid potential ambiguity, do not use the term as a synonym for although or though in documentation.

https://www.ibm.com/docs/en/ibm-style?topic=word-usage#while

> [NOTE]

Reviewer: In the hidden sentence on line 96, there is a spelling error:
> <1> Topology keys from vSphere `openshift-zone` and `openshift-region` catagories.

change "catagories" to "categories"
